examples/Improved_Vis.ipynb

```diff
@@ -0,0 +1,156 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
```

Comment:
Should be able to clean up these imports now that you added the code to the library.